install/kubernetes/helm/istio/charts/security/values.yaml

```diff
@@ -14,6 +14,16 @@ citadelHealthCheck: false
 # 90*24hour = 2160h
 workloadCertTtl: 2160h
 
+# Determines Citadel default behavior if the ca.istio.io/env or ca.istio.io/override
+# namespace are not found on a given namespace.
+#
+# For example: consider a namespace called "target", which has neither the "ca.istio.io/env"
+# nor the "ca.istio.io/override" namespace labels. To decide whether or not to generate secrets
+# for service accounts created in this "target" namespace, Citadel will defer to this option. If the value
+# of this option is "true" in this case, secrets will be generated for the "target" namespace.
+# If the value of this option is "false" Citadel will not generate secrets upon service account creation.
+generateSecretsForUnlabeledNamespaces: true
```

Comment:
Doesn't match the deployment. 

I think we can keep the original name ( better to be consistent with a bad name than have inconsistent and bad names)..